kotlin/find-route/src/main/java/com/esri/arcgisruntime/sample/findroute/MainActivity.kt

```diff
@@ -0,0 +1,321 @@
+            )
+
+            // when the user taps a maneuver, set the viewpoint to that portion of the route
+            onItemClickListener =
+              AdapterView.OnItemClickListener { _, _, position, _ ->
+                // remove any graphics that are not the two stops and the route graphic
+                if (graphicsOverlay.graphics.size > 3) {
+                  graphicsOverlay.graphics.removeAt(graphicsOverlay.graphics.size - 1)
+                }
+
+                // set the viewpoint to the selected maneuver
+                val geometry = directions[position].geometry
+                mapView.setViewpointAsync(
+                  Viewpoint(geometry.extent, 20.0),
+                  1f
+                )
+
+                // create a graphic with a symbol for the maneuver and add it to the graphics overlay
+                val selectedRouteSymbol = SimpleLineSymbol(
+                  SimpleLineSymbol.Style.SOLID,
+                  Color.GREEN, 5f
+                )
+                Graphic(geometry, selectedRouteSymbol).also {
+                  graphicsOverlay.graphics.add(it)
+                }
+
+                // collapse the bottom sheet
+                bottomSheetBehavior.state = BottomSheetBehavior.STATE_COLLAPSED
+              }
+          }
+
+          // when the route is solved, hide the FAB
+          directionFab.visibility = View.GONE
+        } else {
+          val message = "Error loading route parameters."
+          Log.e(TAG, message)
+          Toast.makeText(this, message, Toast.LENGTH_LONG).show()
+        }
+      } catch (e: Exception) {
+        val message = "Error creating route parameters: ${e.message}"
+        Log.e(TAG, message)
+        Toast.makeText(this, message, Toast.LENGTH_LONG).show()
+      }
+    }
+  }
+
+  /**
+   * Set up the source, destination and route symbols
+   */
+  private fun setupSymbols() {
+    //[DocRef: Name=Picture Marker Symbol Drawable-android, Category=Fundamentals, Topic=Symbols and Renderers]
+    // Create a picture marker symbol from an app resource
+    try {
+
+      val startDrawable =
+        ContextCompat.getDrawable(this, R.drawable.ic_source) as BitmapDrawable?
+
+      val pinSourceSymbol = PictureMarkerSymbol.createAsync(startDrawable).get()
+      pinSourceSymbol.apply {
+        loadAsync()
+        addDoneLoadingListener {
+          // add a new graphic as start point
+          val sourcePoint = Point(
+            -117.15083257944445,
+            32.741123367963446,
+            SpatialReferences.getWgs84()
+          )
+          Graphic(sourcePoint, this).also { pinSourceGraphic ->
+            graphicsOverlay.graphics.add(pinSourceGraphic)
+          }
+        }
+        offsetY = 20f
+      }
+    } catch (e: Exception) {
+      val message = "Error loading picture marker symbol: ${e.message}"
+      Log.e(TAG, message)
+      Toast.makeText(this, message, Toast.LENGTH_LONG).show()
+    }
+    //[DocRef: END]
```

Comment:
this and other occurrences of it can be removed assuming we're not using doc refs in the kotlin samples